docs/update.md

```diff
@@ -1,58 +1,82 @@
+    If you run the Operator on [Red Hat Marketplace :octicons-link-external-16:](https://marketplace.redhat.com) or you run Red Hat certified Operators on [OpenShift :octicons-link-external-16:](https://www.redhat.com/en/technologies/cloud-computing/openshift), you need to do additional steps during the upgrade. See [this HOWTO](update_openshift.md) for details.
 
 The list of recommended upgrade scenarios includes two variants:
 
 * Upgrade to the new versions of the Operator *and* Percona XtraDB Cluster,
 * Minor Percona XtraDB Cluster version upgrade *without* the Operator upgrade.
 
-## Upgrading the Operator and CRD
+## Upgrade the Operator and CRD
 
-!!! note
+### Considerations
 
-    The Operator supports **last 3 versions of the CRD**, so it is technically
-    possible to skip upgrading the CRD and just upgrade the Operator. If the CRD
-    is older than the new Operator version *by no more than three releases*, you
-    will be able to continue using the old CRD and even carry on Percona XtraDB
-    Cluster minor version upgrades with it. But the recommended way is to update
-    the Operator *and* CRD.
+1. The Operator version has three digits separated by a dot (`.`) in the format `major.minor.patch`. Here's how you can understand the version `1.17.0`:
 
-Only the incremental update to a nearest version of the
-Operator is supported (for example, update from 1.4.0 to 1.5.0). To update
-to a newer version, which differs from the current version by more
-than one, make several incremental updates sequentially.
+    * `1` is the major version 
+    * `17` is the minor version
+    * `0` is the patch version.     
 
-!!! warning
+    You can only upgrade the Operator to the nearest `major.minor` version (for example, from `1.16.x` to `1.17.x`).     
+
+    If the your current Operator version and the version you want to upgrade to differ by more than one minor version, you need to upgrade step by step. For example, if your current version is `1.15.x` and you want to move to `1.17.x`, first upgrade to `1.16.x`, then to `1.17.x`.    
+
+    Patch versions don't influence the upgrade, so you can safely move from `1.16.1` to `1.17.0`. 
+
+    Check the [Release notes index](ReleaseNotes/index.md) for the list of the Operator versions.
+
+2. CRD supports the **last 3 minor versions of the Operator**. This means it is
+compatible with the newest Operator version and the two older minor versions.
+If the Operator is older than the CRD *by no more than two versions*, you
+should be able to continue using the old Operator version.
+But updating the CRD *and* Operator is the **recommended path**. 
+
+3. Starting with version 1.12.0, the Operator no longer has a separate API version for each release in CRD. Instead, the CRD has the API version `v1`. Therefore, if you run the Operator **older than 1.12.0**, you must update the API version in the CRD manually to run the upgrade. 
```

Comment:
this is only needed if user has an operator installation starting from a release before 1.12.0. so even if you have 1.16.0 operator running in the cluster, if you installed CRDs first from a version before 1.12.0 you will have this problem. to verify you can run `kubectl get crd perconaxtradbclusters.pxc.percona.com -o yaml | yq .status.storedVersions`. if you have any other version than `v1` in the list, you need to do this manual operation.

Reply:
we also need to document the manual operation:

```
$ kubectl proxy & 
$ curl \
	--header "Content-Type: application/json-patch+json" \
	--request PATCH \
	--data '[{"op": "replace", "path": "/status/storedVersions", "value":["v1"]}]' \
http://localhost:8001/apis/apiextensions.k8s.io/v1/customresourcedefinitions/perconaxtradbclusters.pxc.percona.com/status
```

Reply:
This command didn't work for me. It fails with the error: 
`[1] 37530
curl: (7) Failed to connect to localhost port 8001 after 1 ms: Couldn't connect to server
Starting to serve on 127.0.0.1:8001`